src/njoy21/input/THERMR/Card3/Tempr.hpp

```diff
@@ -0,0 +1,41 @@
+struct Tempr {
+  using Value_t = std::vector< Quantity< Kelvin > >;
+  static std::string name(){ return "tempr"; }
+
+  static std::string description() {
+    return 
+      "The tempr argument is a list of ntemp temperatures (where ntemp was\n"
+      "defined in card2). These tempr inputs are the temperatures at which\n"
+      "the thermal scattering cross sections are evaluated. Each\n"
+      "temperature is in kelvin, and therefore must be positive.";
+  }
+
+  static bool verify( const Value_t& temps,
+                      const Argument< THERMR::Card2::Ntemp > & ntemp ){
+    if ( long(temps.size()) != ntemp.value ){
```

Comment:
The vector parser does this before the verification is called. This check is redundant.